app/components/menubar/newMenuBar.js

```diff
@@ -0,0 +1,350 @@
+import Cookies from "js-cookie";
+import Link from "next/link";
+import { DummyLoginButton } from "../auth/dummy";
+import RCGoogleLoginButton from "../auth/goauth/ui/GoogleRCLogin";
+
+const userCookie = Cookies.get("user");
+const isUserLoggedIn = Cookies.get("hashmail") ? true : false;
+const hasAllRequiredCreds =
+  process.env.NEXT_PUBLIC_GOOGLE_CLIENT_ID &&
+  process.env.NEXT_PUBLIC_RC_URL;
+
+const CustomToggle = forwardRef(({ children, onClick }, ref) => (
+  <a
+    className={styles.elipses}
+    href=""
+    ref={ref}
+    onClick={(e) => {
+      e.preventDefault();
+      onClick(e);
+    }}
+  >
+    {children}
+    <span className={styles.threedots} />
+  </a>
+));
+
+if (!hasAllRequiredCreds) console.log("RC4Community is now using a dummy Auth Component! If you wish to use a robust Auth component, provide all the credentials first (https://github.com/RocketChat/RC4Community/tree/master/app/components/auth)")
+
+
+const ArrowIcon = () => {
+  return (
+    <svg
+      width='14'
+      height='15'
+      viewBox='0 0 32 17'
+      fill='none'
+      xmlns='http://www.w3.org/2000/svg'
+    >
+      <path d='M1.5 1.5L15.5 15.5L31 1.5' stroke='black' strokeWidth='2' />
+    </svg>
+  );
+};
+
+const MobileNav = ({ nav_Items }) => {
+  const [dropDown, setDropDown] = useState({ show: false, _id: 0 });
+  return (
+    <Navbar className='d-lg-none' expand={false}>
+      <Container fluid>
+        <Navbar.Toggle
+          aria-controls='offcanvasNavbar'
+          className={styles.default_toggler}
+        >
+          <div
+            className={`${styles.navbar_toggler} navbar-toggler collapsed d-flex d-lg-none flex-column justify-content-around bg-white`}
+            type='button'
+          >
+            <span className={`${styles.toggler_icon} mb-2`}></span>
+            <span className={`${styles.toggler_icon} mt-2`}></span>
+          </div>
+        </Navbar.Toggle>
+        <Navbar.Offcanvas
+          id='offcanvasNavbar'
+          aria-labelledby='offcanvasNavbarLabel'
+          placement='start'
+        >
+          <Offcanvas.Header closeButton>
+            <Navbar.Brand
+              href='/'
+              className='d-flex justify-content-center align-items-center '
+            >
+              <BrandLogo
+                logoLink={
+                  "https://global-uploads.webflow.com/611a19b9853b7414a0f6b3f6/611bbb87319adfd903b90f24_logoRC.svg"
+                }
+                imageTitle={"Rocket.Chat"}
+                brandName={"Rocket.Chat Community"}
+                height={30}
+                width={132}
+              />
+            </Navbar.Brand>
+          </Offcanvas.Header>
+          <Offcanvas.Body>
+            {nav_Items?.map((nav_Item) =>
+              nav_Item.url ? (
+                <div key={nav_Item.id}>
+                  <Row
+                    className={`${styles.dropdown} d-flex flex-row justify-content-between align-items-center mt-3 `}
```

Comment:
This Error occurs when opening the Nav items

![image](https://user-images.githubusercontent.com/61188295/213871346-a555c626-f81c-4f97-98c5-01ee6b459bcb.png)